rpc/cc/arduino/cli/files/v1/files.proto

```diff
@@ -0,0 +1,45 @@
+// This file is part of arduino-cli.
+//
+// Copyright 2020 ARDUINO SA (http://www.arduino.cc/)
+//
+// This software is released under the GNU General Public License version 3,
+// which covers the main part of arduino-cli.
+// The terms of this license can be found at:
+// https://www.gnu.org/licenses/gpl-3.0.en.html
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to
+// modify or otherwise use the software for commercial activities involving the
+// Arduino software without disclosing the source code of your own applications.
+// To purchase a commercial license, send an email to license@arduino.cc.
+
+syntax = "proto3";
+
+package cc.arduino.cli.files.v1;
+
+option go_package = "github.com/arduino/arduino-cli/rpc/cc/arduino/cli/files/v1;files";
+
+// Arduino Files service API
+service FilesService {
+  // Load file content from the file system
+  rpc LoadFile(LoadFileRequest) returns (LoadFileResponse) {}
+}
+
+enum ContentType {
+  CONTENT_TYPE_RAW_UNSPECIFIED = 0;
+  CONTENT_TYPE_ZIP = 1;
+}
+
+message LoadFileRequest {
+  // path as it's returned in LoadSketchResponse for instance
+  string path = 1;
+  // Optional
+  ContentType type = 2;
+}
+
+message LoadFileResponse {
+  // File content
+  bytes content = 1;
```

Comment:
or it could be smth like (depends on the design if whether we want to let the client know error code to add custom behaviour):
```
oneof {
    bytes content = 1;
    grpc.status error =2;
}